devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -101,11 +115,12 @@ def zwaveEvent(physicalgraph.zwave.Command cmd) {
 }
 
 def setButton(button) {
+	log.debug "button: $button"
 	switch(button) {
-		case "open":
+		case "statelessCurtainPowerButton_open_button":
 			open()
 			break
-		case "close":
+		case "statelessCurtainPowerButton_close_button":
```

Comment:
These aren't the correct values for this capability. They should be "open" and "close". The capability's command definition is as follows:
```
commands:
  setButton:
    arguments:
    - name: button
      required: true
      schema:
        type: string
        enum:
          - open
          - close
          - pause
      type: ENUM
      values:
        - open
        - close
        - pause
```

If this value is difference then we should investigate why they are.

Reply:
@dkirker - When i started working with Nano Shutter using the default device handler, it wasn't working initially.

I soon found that all values were parsing to Pause() only.

I used log.debug "button: $button" (shown on line 118) to find the states;

"statelessCurtainPowerButton_open_button" and "statelessCurtainPowerButton_close_button" which were reporting when pressing open or close from SmartThings connect.

If you run my code, you should be able to use line 118 log.debug to see the specific state that is reported.

Reply:
@Aeotec-ccheng That's what I was suspecting had happened. Thanks for confirming. Let's go ahead and do the following so that when the values that should be sent are sent the code will still work:
```
		case "open":
		case "statelessCurtainPowerButton_open_button":
			open()
			break
		case "close":
		case "statelessCurtainPowerButton_close_button":
			close()
			break
```

Reply:
@dkirker - Thanks for the coding recommendation, added that in to the new commit.

Reply:
@Aeotec-ccheng Awesomesauce!